Modules/Page/Http/Controllers/PublicController.php

```diff
@@ -96,12 +98,13 @@ private function getTemplateForPage($page)
     }
 
     /**
-     * Throw a 404 error page if the given page is not found or draft
+     * Throw a 404 error page if the given page is not found
+     *
      * @param $page
      */
     private function throw404IfNotFound($page)
     {
-        if (null === $page || $page->status === $this->disabledPage) {
+        if (is_null($page)) {
```

Comment:
Why did you remove the check for a disabled page? If a page's status is set to `false`, we don't want someone to be able to access it at all.

Reply:
Rally don't no, It was mu default controller, I didn't removed any of them